Control/DeepSeq.hs

```diff
@@ -209,6 +209,13 @@ f $!! x = x `deepseq` f x
 force :: (NFData a) => a -> a
 force x = x `deepseq` x
 
+-- | Deeply strict version of 'Control.Applicative.<$>'.
+--
+-- @since 1.4.3.0
+(<$!!>) :: (Monad m, NFData b) => (a -> b) -> m a -> m b
```

Comment:
[This doesn't appear to be exported from `Control.DeepSeq`:](https://travis-ci.org/haskell/deepseq/jobs/166651189#L306)

```
Control/DeepSeq.hs:216:3: warning: [-Wunused-top-binds]
    Defined but not used: ‘<$!!>’
```